x-pack/test/rule_registry/security_and_spaces/tests/basic/get_alert_by_id.ts

```diff
@@ -0,0 +1,219 @@
+  /** Unauthorized users */
+  unauthorizedUsers: User[];
+}
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext) => {
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  const esArchiver = getService('esArchiver');
+
+  const TEST_URL = '/internal/rac/alerts';
+  const ALERTS_INDEX_URL = `${TEST_URL}/index`;
+  const SPACE1 = 'space1';
+  const SPACE2 = 'space2';
+  const APM_ALERT_ID = 'NoxgpHkBqbdrfX07MqXV';
+  const APM_ALERT_INDEX = '.alerts-observability-apm';
+  const SECURITY_SOLUTION_ALERT_ID = '020202';
+  const SECURITY_SOLUTION_ALERT_INDEX = '.alerts-security-solution';
+
+  const getAPMIndexName = async (user: User) => {
+    const {
+      body: indexNames,
+    }: { body: { index_name: string[] | undefined } } = await supertestWithoutAuth
+      .get(`${getSpaceUrlPrefix(SPACE1)}${ALERTS_INDEX_URL}`)
+      .auth(user.username, user.password)
+      .set('kbn-xsrf', 'true')
+      .expect(200);
+    const observabilityIndex = indexNames?.index_name?.find(
+      (indexName) => indexName === APM_ALERT_INDEX
+    );
+    expect(observabilityIndex).to.eql(APM_ALERT_INDEX); // assert this here so we can use constants in the dynamically-defined test cases below
+  };
+
+  const getSecuritySolutionIndexName = async (user: User) => {
+    const {
+      body: indexNames,
+    }: { body: { index_name: string[] | undefined } } = await supertestWithoutAuth
+      .get(`${getSpaceUrlPrefix(SPACE1)}${ALERTS_INDEX_URL}`)
+      .auth(user.username, user.password)
+      .set('kbn-xsrf', 'true')
+      .expect(200);
+    const securitySolution = indexNames?.index_name?.find(
+      (indexName) => indexName === SECURITY_SOLUTION_ALERT_INDEX
+    );
+    expect(securitySolution).to.eql(SECURITY_SOLUTION_ALERT_INDEX); // assert this here so we can use constants in the dynamically-defined test cases below
+  };
+
+  describe('Alerts - GET - RBAC - spaces', () => {
+    before(async () => {
+      await getSecuritySolutionIndexName(superUser);
+      await getAPMIndexName(superUser);
+
+      await esArchiver.load('x-pack/test/functional/es_archives/rule_registry/alerts');
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/rule_registry/alerts');
+    });
+
+    it('should return a 404 when superuser accesses not-existent alert', async () => {
+      await supertestWithoutAuth
+        .get(`${getSpaceUrlPrefix()}${TEST_URL}?id=myfakeid&index=${APM_ALERT_INDEX}`)
+        .auth(superUser.username, superUser.password)
+        .set('kbn-xsrf', 'true')
+        .expect(404);
+    });
+
+    it('should return a 404 when superuser accesses not-existent alerts as data index', async () => {
+      await supertestWithoutAuth
+        .get(`${getSpaceUrlPrefix()}${TEST_URL}?id=${APM_ALERT_ID}&index=myfakeindex`)
+        .auth(superUser.username, superUser.password)
+        .set('kbn-xsrf', 'true')
+        .expect(404);
+    });
```

Comment:
Nit: you can remove the superuser-specific tests here and add them to the group for all users instead (see comment below)

```suggestion
```